lib/tasks/export.rake

```diff
@@ -165,12 +164,9 @@ namespace :export do
     scope = scope.where("editions.updated_at >= ?", ENV["FROM"]) if ENV["FROM"]
     total = scope.count
 
-    document_data = scope.find_each.with_index.map do |document, index|
-      puts "#{index + 1}/#{total} exported" if ((index + 1) % 100).zero?
-      ExportNewsDocument.new(document).as_json
+    scope.find_each.with_index.each do |document, index|
+      puts "#{index + 1}/#{total} exported", STDERR if ((index + 1) % 100).zero?
+      puts ExportNewsDocument.new(document).as_json.to_json
```

Comment:
Seems a bit odd to chain `as_json.to_json`?

Reply:
Yeah, it's because `as_json` creates a hash, which `to_json` then converts to actual JSON. The naming is confusing - I could change it to `JSON.dumps(... .as_json)` if you want?

Reply:
I'm not sure it's any better, so will stop blocking this PR and approve 👍